doc/build-windows.md

```diff
@@ -68,7 +68,7 @@ To build executables for Windows 64-bit, install the following dependencies:
 Then build using:
 
     cd depends
-    make HOST=x86_64-w64-mingw32
+    make HOST=x86_64-w64-mingw32 PATH=$(getconf PATH)
```

Comment:
Does it also interfere with the other `make` commands used to build/install bitcoin core itself?